test/index.js

```diff
@@ -260,8 +262,7 @@ describe('node-lkl', function() {
 			it('non existent file', function(done) {
 				lkl.fs.access(this.doesNotExist, function(err) {
 					assert.notEqual(err, null, 'error should exist');
-					// assert.strictEqual(err.code, 'ENOENT');
-					assert.strictEqual(err.path, this.doesNotExist);
```

Comment:
This test  was broken: `err.path` was undefined and `this.doesNotExist` was undefined too (wrong `this`).